document/js-api/index.bs

```diff
@@ -788,9 +788,10 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
     1. Let |store| be the current agent's [=associated store=].
     1. Let |globaladdr| be |global|.\[[Global]].
     1. Let |globaltype| be [=type_global=](|store|, |globaladdr|).
-    1. If |globaltype| is of the form [=const=] |type|, throw a {{TypeError}.
-    1. If |type| is [=𝗂𝟨𝟦=], throw a {{TypeError}}.
-    1. Let |value| be [=ToWebAssemblyValue=](|v|, |type|).
+    1. Assert: |globaltype| is of the form |mut| |valuetype|.
```

Comment:
Nit: this is just a pattern match, not an assertion, since it holds by construction.